src/katgpucbf/xbgpu/engine.py

```diff
@@ -183,21 +180,23 @@ async def gpu_proc_loop(self) -> None:
         """Perform all GPU processing of received data in a continuous loop.
 
         This function should do the following:
-        * Get an RxQueueItem off the rx_item_queue
-        * Ensure it is not a NoneType value (indicating shutdown sequence)
-        * await any outstanding events associated with the RxQueueItem
-        * TODO: Figure out whether the `is not None` check on the Chunk is necessary
-        * Apply GPU processing to data in the RxQueueItem
-            * Bind input buffer(s) accordingly
-        * Obtain a free TxQueueItem of the tx_free_item_queue
-            * Add event marker to wait for the proc_command_queue
-            * Put the prepared TxQueueItem on the tx_item_queue
```

Comment:
Need a blank line here - it's not rendering properly. Might also need a blank line before "- Add event marker..."